src/lib/constant_mappings.cpp

```diff
@@ -35,9 +35,28 @@ const boost::bimap<PredicateCondition, std::string> predicate_condition_to_strin
         {PredicateCondition::IsNotNull, "IS NOT NULL"},
     });
 
+const boost::bimap<PredicateCondition, std::string> predicate_condition_to_string_readable =
```

Comment:
This has nothing to do with your change directly. I don’t like the redundancy this extra bitmap adds. But we have that problem every time we build a switch case construct around a PredicateCondition. I would prefer a more object oriented approach.

Reply:
We needed a bitmap like that, because our tests and benchmarks need a mapping from predicate to text, because test and benchmark names are auto-generated and may only consist of letters and numbers (e.g., no `<=`). That means we definitely need that mapping somewhere.

Reply:
I just asked myself if it wouldn't be smarter in general to store that information within the PredicateCondition.